R/install_livy.R

```diff
@@ -0,0 +1,93 @@
+#' Install Livy
+#'
+#' Automatically download and install \href{http://livy.io/}{\samp{livy}}.
+#' \samp{livy} provides a REST API to Spark.
+#'
+#' @param spark_home The path to a Spark installation. The downloaded and
+#'   installed version of \samp{livy} will then be associated with this Spark
+#'   installation. When unset (\samp{NULL}), the value is inferred based on
+#'   the value of \samp{spark_version} supplied.
+#' @param spark_version The version of Spark to use. When unset (\samp{NULL}),
+#'   the value is inferred based on the value of \samp{livy_version} supplied.
+#'   A version of Spark known to be compatible with the requested version of
+#'   \samp{livy} is chosen when possible.
+#' @param livy_version The version of \samp{livy} to be installed.
+#' @export
+livy_install <- function(spark_home    = NULL,
```

Comment:
But `version` as a placeholder for the Spark version makes sense in the context of Spark functions -- here, I think users would infer that `version` is associated with the `livy` version to be installed.

For an installation function, IMHO it makes more sense to request the user either explicitly provide a Spark version (in which case we'll construct a Spark home path for them), or a whole Spark home (in which case we assume the user knows what they're doing)